src/helpers/arcHelpers.ts

```diff
@@ -0,0 +1,103 @@
+import { cartesian2DDistance, getMidPoint } from '@vcmap/core'
+import { Cartesian3, CatmullRomSpline } from '@vcmap/cesium'
+import type { Coordinate } from 'ol/coordinate'
+import type { TravelTimeModel } from '@/model/travel-time.model'
+import { LineString, Point } from 'ol/geom'
+import { lineStringsFromTraveltimes } from '@/helpers/traveltimesHelper'
+import { Feature } from 'ol'
+import type { RennesApp } from '@/services/RennesApp'
+import { Cartesian2 } from '@vcmap-cesium/engine'
+
+export const arcFactor = 0.15
+
+function getArcCoordinates(
```

Comment:
Perhaps we should put a note that we took the function from vc map core. Just in case someone found a bug here, they can see if if it's fixed or not in the original code :D